sections/06_discussion.md

```diff
@@ -68,46 +72,46 @@ interpretable healthcare features from trained deep models.
 
 ### Data limitations
 
-*Related to evaluation, are there data quality issues in genomic, clinical, and
-other data that make this domain particularly challenging?  Are these worse than
-what is faced in other non-biomedical domains?*
-
-*Many applications have used relatively small training datasets.  We might
-discuss workarounds (e.g. semi-synthetic data, splitting instances, etc.) and
-how this could impact future progress.  Might this be why some studies have
-resorted to feature engineering instead of learning representations from low-
-level features?  Is there still work to be done in finding the right low-level
-features in some problems?*
-
-###### Biomedical data is often "Wide"
-
-*Biomedical studies typically deal with relatively small sample sizes but each
-sample may have millions of measurements (genotypes and other omics data, lab
-tests etc).*
-
-*Classical machine learning recommendations were to have 10x samples per number
-of parameters in the model.*
-
-*Number of parameters in an MLP. Convolutions and similar strategies help but do
-not solve*
-
-*Bengio diet networks paper*
-
+A lack of large-scale, high-quality, labeled training data has impacted deep
+learning in nearly all applications we have discussed, from healthcare to
+genomics to drug discovery.  The challenges of training complex, high-parameter
+neural networks from few examples are obvious, but uncertainty in the labels of
+training instances can be just as problematic.  In genomics, for example,
+labeled data may be derived from an experimental assay that has known and
+unknown technical artifacts, biases, and error profiles.  It is possible to
+weight training examples or construct Bayesian models to account for uncertainty
```

Comment:
I made #377 for you to add these lines.  You'll do a better job of getting the example correct.

This is along the lines of what I meant by "weight training examples", but I didn't explain that phrase well.